docs/articles/user-guide/source-format.md

```diff
@@ -4,6 +4,142 @@ title: Source Format
 
 The commands that Salesforce Extensions for VS Code uses to push, pull, deploy, and retrieve your source assume that your files are in source format (rather than metadata format). Source format is optimized for working with version control systems. For details, see [Salesforce DX Project Structure and Source Format](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_source_file_format.htm) in the _Salesforce DX Developer Guide_.
 
-Because Force.com IDE used metadata format, you can’t open your Force.com IDE projects in VS Code. To work with the commands described in this article, either convert your metadata to source format (using `sfdx force:mdapi:convert`) or create a new project and then retrieve the metadata from your org using the manifest (`package.xml` file) that you used in your previous IDE.
+Because Force.com IDE used metadata format, you can’t directly open your Force.com IDE projects in VS Code. You must either convert your metadata to source format (using `sfdx force:mdapi:convert`) or create a new project and then retrieve the metadata from your org using the manifest (`package.xml` file) that you used in your previous IDE.
 
-For information about converting to source format and maintaining Git history, see [this blog post](https://ntotten.com/2018/05/11/convert-metadata-to-source-format-while-maintain-git-history/).
+## Convert Metadata to Source Format and Maintain Git History
+
+If you have a massive Salesforce project that is in metadata format and tracked in Git, a bulk convert to the new source format loses all the revision history. This is because Git has built in limits and it fails to detect the enormous amount of changes that happen at the same time. The solution is to convert the project to source format in smaller chunks so that you can maintain the revision history. Let’s take the [dreamhouse](https://github.com/dreamhouseapp/dreamhouse-sfdx) project as example to follow the conversion steps.
```

Comment:
Massive is an ambiguous word. Lets either remove it or be more clear.

Reply:
Removed 'massive'.